server/sonar-web/src/main/js/apps/projects/store/actions.js

```diff
@@ -49,15 +49,13 @@ const onFail = dispatch => error => {
 };
 
 const onReceiveMeasures = (dispatch, projects) => response => {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "projects". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)